app/models/user/preferences.rb

```diff
@@ -0,0 +1,11 @@
+class User::Preferences < ApplicationRecord
```

Comment:
I considered `Preferences` or `Settings` or `GeneralPreferences` and eventually went for this.